Dan Tudor/05-3.10/main.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+void nrOfBacknotes(int n, int *nr)
+{
+    if(n/500 > 0)
```

Comment:
What if you'd put all the banknotes values in an array and then iterate through that array instead of writing so much duplicate code?